### PR TITLE
User Warning to Morrowind Plugin Limit

### PR DESCRIPTION
**Pull request by BraniganMatthew**
Warns users exceeding the normal plugin limit for Morrowind, reminding them to not open the game without using the script extender.

### CODE REVIEW

File: src/gui/state/game/game.cpp [hunk 1]
```diff
@@ -720,7 +720,20 @@ std::vector<Message> Game::GetMessages() const {
...
+      auto logger = getLogger();
+    if (logger) {
+      logger->warn(
+          "More than 255 normal plugins are activated at the same time.");
+    }
+    output.push_back(PlainTextMessage(
+        MessageType::warn,
+        boost::locale::translate(
+          "Do not launch Morrowind without the use of MWSE or it will"
+          "cause unaccountable damage to your game.")));
```
Comment (Ortham) [inline src/gui/state/game/game.cpp:734]: I'm a bit confused by the use of `unaccountable` here, I don't know what it's trying to communicate (though I'm not a Morrowind mod user, so maybe that's why). Does it mean that it won't obvious why things break when they do?

**Comment by Ortham** [comment]
@BraniganMatthew am I right in thinking that this PR answers the question in my last comment in #1558?

**Comment by BraniganMatthew** [comment]
@Ortham Sorry for the delay, this PR is my way of answering your question on that issue. I believe a warning is best for this problem because a user with that number of mods should know the risks of going past that 256 limit. Also, the word "unaccountable" is meant to describe severity.

**Comment by Ortham** [comment]
@BraniganMatthew OK, thanks. Could you replace "unaccountable" with "severe"? I think that would be clearer, especially for ESL readers, and it would probably help translators too.

Otherwise, I think I'm happy to merge this.

**Comment by BraniganMatthew** [comment]
@Ortham Ok just pushed that change

File: src/gui/state/game/game.cpp [hunk 2]
```diff
@@ -720,7 +720,20 @@ std::vector<Message> Game::GetMessages() const {
 
   static constexpr size_t SAFE_MAX_ACTIVE_NORMAL_PLUGINS = 254;
 
-  if (activeNormalPluginsCount > SAFE_MAX_ACTIVE_NORMAL_PLUGINS &&
+  if (activeNormalPluginsCount > SAFE_MAX_ACTIVE_NORMAL_PLUGINS && settings_.Type() == GameType::tes3 && 
+      std::filesystem::exists(preludePath_/"MWSE.dll")) {
+      auto logger = getLogger();
+    if (logger) {
+      logger->warn(
+          "More than 255 normal plugins are activated at the same time.");
```
Comment (Ortham) [inline src/gui/state/game/game.cpp:728]: I just spotted that the message says more than 255 but the check is for more than 254.

**Review by Ortham — CHANGES_REQUESTED** [review]

**Comment by BraniganMatthew** [comment]
@Ortham I fixed the plugin count.

**Review by Ortham — APPROVED** [review]

**Comment by Ortham** [comment]
Thanks!

**Pull request merged**